### PR TITLE
Added views module

### PR DESCRIPTION
**Pull request by khoroshevskyi**

### CODE REVIEW

File: pepdbagent/modules/view.py [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+
+        :param namespace: namespace of the project
+        :param name: name of the project
+        :param tag: tag of the project
+        :param view_name: name of the view
+        :param sample_name: sample name
+        :return: None
+        """
+        if isinstance(sample_name, str):
+            sample_name = [sample_name]
```
Comment (nleroy917) [inline pepdbagent/modules/view.py:240]: I like this approach I do this frequently 👍🏻

**Review by nleroy917 — APPROVED** [review]
Looks good, might need a refresher on the benefit of views, but the code seems reasonable across the board 👍🏻

File: pepdbagent/__init__.py [hunk 2]
```diff
@@ -2,8 +2,11 @@
 import coloredlogs
 import logmuse
 
-from ._version import __version__
-from .pepdbagent import *
+from pepdbagent._version import __version__
+from pepdbagent.pepdbagent import PEPDatabaseAgent
```
[the remark this answers is not in the document]
Comment (khoroshevskyi) [reply pepdbagent/__init__.py:6]: yeah, but pepdbagent was from very begining written in that way ...
Wanted to be consistent in this package

File: pepdbagent/models.py [hunk 3]
```diff
@@ -160,3 +160,27 @@ class ProjectRegistryPath(BaseModel):
     namespace: str
     name: str
     tag: str = DEFAULT_TAG
+
+
+class ViewAnnotation(BaseModel):
```
[the remark this answers is not in the document]
Comment (khoroshevskyi) [reply pepdbagent/models.py:165]: I will work more on this tomorrow. for me, it is still a little bit immature (not sure if this is correct word in english haha)

**Pull request merged**